storage/innobase/srv/srv0start.cc

```diff
@@ -2581,18 +2581,30 @@ dberr_t srv_start(bool create_new_db) {
 
     srv_dict_metadata = recv_recovery_from_checkpoint_finish(*log_sys, false);
 
+    // Disabled for the clone cross-engine synchronization: the metadata updates
```

Comment:
Is this true in the case we don't have a MyRocks setup. Is it true that as long as the engine synchronization is using pfs.log_status, then this code can be skipped. Is this correct?

Reply:
The old code works when clone LSN == redo log LSN
The new code works when clone LSN <= redo log LSN

Thus the new code works with or without pfs.log_status synchronization